src/platform/library/include/platform/lib/memory.h

```diff
@@ -29,6 +29,10 @@
 
 #define SHARED_DATA
 
+#define uncache_to_cache(address)	(address)
+#define cache_to_uncache(address)	(address)
+#define is_uncached(address)		0
```

Comment:
Aren't these supposed to be platform specific now?!

Reply:
@paulstelian97 There are still couple of occurrences in generic code like in mailbox header. That's why it's still needed. It will be refactored soon.

Reply:
Alright, guess I can wait for that.